MaveDB.pm

```diff
@@ -267,7 +268,7 @@ sub run {
     # Check if transcripts match
     if ($self->{transcript_match}) {
       my $transcript = $_->{result}->{MaveDB_refseq};
-      next unless $self->_transcripts_match($tva, $transcript);
+      next unless $transcript eq '' or $self->_transcripts_match($tva, $transcript);
```

Comment:
You should also update the check in `_transcript_match` [here](https://github.com/Ensembl/VEP_plugins/blob/ed13b43a41058860da78fcab07a47325aed08eaa/MaveDB.pm#L207) -

```
my @refseq = split(/,/, $tr->{_refseq}) if defined $tr->{_refseq} and $tr->{_refseq} ne '-';
```

Some transcripts may not have refseq and it produces a lot of warnings.

Reply:
Fixed! Thanks